README.md

```diff
@@ -148,12 +148,13 @@ The **Options** schema allows certain options within the report to be toggled on
 
 The **InfoLevel** schema allows configuration of each section of the report at a granular level. The following sections can be set.
 
-There are 2 levels (0-1) of detail granularity for each section as follows;
+There are 4 levels (0-3) of detail granularity for each section as follows;
 
 | Setting | InfoLevel         | Description                                                                                                                                |
 |:-------:|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
 |    0    | Disabled          | Does not collect or display any information                                                                                                |
 |    1    | Enabled | Provides condensed, detailed information for a collection of objects                                                                                 |
+|    2    | Adv Summary       | Provides condensed, detailed information for a collection of objects                                                                       |
```

Comment:
InfoLevel 2 description is exactly the same as InfoLevel 1,

Here's how i normally do it 

| Setting | InfoLevel         | Description                                                                                                                                |
|:-------:|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
|    0    | Disabled          | Does not collect or display any information                                                                                                |
|    1    | Enabled / Summary | Provides summarised information for a collection of objects                                                                                |
|    2    | Adv Summary       | Provides condensed, detailed information for a collection of objects

Reply:
Done!